### PR TITLE
adds twilio sample

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: appengine/standard/twilio/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Twilio & Google App Engine
+
+This sample application demonstrates how to use [Twilio with Google App Engine](https://cloud.google.com/appengine/docs/php/sms/twilio).
+
+## Setup
+
+Before running this sample:
+
+1. You will need a [Twilio account](http://www.mailgun.com/google).
```
Comment (tmatsuo) [inline appengine/standard/twilio/README.md]: mailgun?

File: appengine/standard/twilio/app.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?php
+
```
Comment (tmatsuo) [inline appengine/standard/twilio/app.php]: Let's add copyright header

File: appengine/standard/twilio/app.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+use Silex\Application;
+
+$app = new Application();
+
+$app->get('/', function () use ($app) {
+    if ($app['twilio.account_sid'] == 'TWILIO_ACCOUNT_SID') {
```
Comment (tmatsuo) [inline appengine/standard/twilio/app.php]: Is it possible to check TWILIO_AUTH_TOKEN too?

Comment (bshaffer) [inline appengine/standard/twilio/app.php]: I don't really feel like this is necessary. It's just an obvious check to see if they've looked at the `index.php`.

File: appengine/standard/twilio/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+1. Update `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN` in `index.php` to match your
+   Twilio credentials. These can be found in your [account settings]
+   (https://www.twilio.com/user/account/settings)
+1. Update `TWILIO_FROM_NUMBER` in `index.php` with a number you have authorized
```
Comment (tmatsuo) [inline appengine/standard/twilio/README.md]: TWILIO_TO_NUMBER too?

File: appengine/standard/twilio/phpunit.xml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  limitations under the License.
+-->
+<phpunit bootstrap="test/bootstrap.php">
+    <testsuites>
+        <testsuite name="PHP twilio test">
+            <directory>test</directory>
+        </testsuite>
+    </testsuites>
+    <logging>
+        <log type="coverage-clover" target="../../../build/logs/clover-twilio.xml"/>
```
Comment (tmatsuo) [inline appengine/standard/twilio/phpunit.xml]: Future todo:

'../../..' for the clover file is error prone

File: appengine/standard/twilio/README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
+# Twilio & Google App Engine
```
Comment (JustinBeckwith) [inline appengine/standard/twilio/README.md]: Anything make this different between standard and flexible?

File: appengine/standard/twilio/app.php (hunk 3 above)
Comment (tmatsuo) [inline appengine/standard/twilio/app.php]: Ok I can live with this

**Comment by bshaffer** [comment]
I can haz LGTM?

**Comment by tmatsuo** [comment]
Yes LGTM

**Pull request merged**